### ISSUE TITLE
Is there any documentation on setting up the AdventureWorks sample model?

### ISSUE THREAD

**Issue by dannief** [NakedObjectsGroup/NakedObjectsFramework#135]
I haven't gone through the whole documentation yet, but for evaluation purposes, I would like to run a more complex example than the Template (which I did get to work).

What I have done so far is add the AdventureWorksModel project to the Template solution. I gather that I need to configure the ```NakedObjectsRunSettings``` class, but I am not sure what needs to be added to the various sections.

Any help would be greatly appreciated

**Comment by richardpawson** [comment]
Well you could copy our demo version of NakedObjectsRunSettings from here:

https://github.com/NakedObjectsGroup/NakedObjectsFramework/blob/master/Demo/NakedObjects.Rest.App.Demo/App_Start/NakedObjectsRunSettings.cs

But please note: if you want to progress with Naked Objects you need to read the Developer Manual (  https://github.com/NakedObjectsGroup/NakedObjectsFramework/blob/master/Documentation/NOF9DeveloperManual.docx ). We put a lot of effort into that, and this forum should not be used as a substitute for reading that.

**Comment by richardpawson** [comment]
Well you could copy our demo version of NakedObjectsRunSettings from here:

https://github.com/NakedObjectsGroup/NakedObjectsFramework/blob/master/Demo/NakedObjects.Rest.App.Demo/App_Start/NakedObjectsRunSettings.cs

But please note: if you want to progress with Naked Objects you need to read the Developer Manual (  https://github.com/NakedObjectsGroup/NakedObjectsFramework/blob/master/Documentation/NOF9DeveloperManual.docx ). We put a lot of effort into that, and this forum should not be used as a substitute for reading that.

**Comment by dannief** [comment]
Thanks. I got the AdventureWorksModel working with the settings class you provided. I appreciate the response.

Just to say though, I am going through the documentation, just have not finished yet. I think it would help with adoption though if someone who is evaluating the framework could at least see a real world application in action without having to go through the whole documentation first. It would be good if help on how to set up and run the AdventureWorks demo was included in the wiki. Then we could read the documentation to understand how it all works and develop of own application.

On another note. I noticed that all the links in the manual are absolute links and so they do not work. Logged issue #136.
